web-client/tests/e2e/globals.js

```diff
@@ -0,0 +1,104 @@
+///////////////////////////////////////////////////////////////////////////////////
+// Refer to the entire list of global config settings here:
+// https://github.com/nightwatchjs/nightwatch/blob/master/lib/settings/defaults.js#L16
+//
+// More info on test globals:
+//   https://nightwatchjs.org/gettingstarted/configuration/#test-globals
+//
+///////////////////////////////////////////////////////////////////////////////////
+
+module.exports = {
+  // this controls whether to abort the test execution when an assertion failed and skip the rest
+  // it's being used in waitFor commands and expect assertions
+  abortOnAssertionFailure: true,
+
+  // this will overwrite the default polling interval (currently 500ms) for waitFor commands
+  // and expect assertions that use retry
+  waitForConditionPollInterval: 500,
+
+  // default timeout value in milliseconds for waitFor commands and implicit waitFor value for
+  // expect assertions
+  waitForConditionTimeout: 5000,
+
+  'default': {
+    /*
+    The globals defined here are available everywhere in any test env
+    */
+
+    /*
+    myGlobal: function() {
+      return 'I\'m a method';
+    }
+    */
+  },
+
+  'firefox': {
+    /*
+    The globals defined here are available only when the chrome testing env is being used
```

Comment:
Not a stopper, but I see the comment says "chrome" when it should read "firefox" -- looks like some copy-paste legacy

Reply:
Please feel free to squash & merge as-is since we can correct the comment in a later commit ... or alternatively update first, up to you